src/DynamoCore/DynamoCore.csproj

```diff
@@ -23,8 +23,8 @@
   <ItemGroup>
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
     <PackageReference Include="DynamoVisualProgramming.Analytics" Version="3.0.0.2506" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.16.0.3176" GeneratePathProperty="true" />
```

Comment:
Are supposed to bump up to 2.17.0?

Reply:
not super important, I can't remember what the versioning strategy is for LibG at the moment.

Reply:
in a separate PR maybe it should just get bumped all the way to 2.18